koku/reporting/migrations/0121_auto_20200728_2258.py

```diff
@@ -56,12 +56,15 @@ class Migration(migrations.Migration):
                 DROP INDEX IF EXISTS ocpall_storage_summary;
                 DROP MATERIALIZED VIEW IF EXISTS reporting_ocpall_storage_summary;
 
+                DROP INDEX IF EXISTS ocpall_cost_daily_summary;
                 DROP INDEX IF EXISTS ocpallcstdlysumm_node;
                 DROP INDEX IF EXISTS ocpallcstdlysumm_node_like;
                 DROP INDEX IF EXISTS ocpallcstdlysumm_nsp;
                 DROP INDEX IF EXISTS ocpall_product_code_ilike;
+
                 DROP MATERIALIZED VIEW IF EXISTS reporting_ocpallcostlineitem_daily_summary;
 
+                DROP INDEX IF EXISTS ocpall_cost_project_daily_summary;
```

Comment:
🤔  Is this allowed? Modifying an old migration?

Reply:
🙊 🤐 😯